docs/dictionary/function/revXMLEvaluateXPath.lcdoc

```diff
@@ -63,7 +63,7 @@ instance, given xml data of
     
 then
 
-    put "/bookstore/book/[price&lt;40]/" into pXPathExpression
+    put "/bookstore/book[price&lt;40]/" into pXPathExpression
     put revXMLEvaluateXPath(pDocID, pXPathExpression)
```

Comment:
@livecodesam there is another instance of the same typo/mistake that needs to be corrected